src/cccid.rs

```diff
@@ -48,16 +48,27 @@ const CCC_TMPL: &[u8] = &[
     0x00, 0xfe, 0x00,
 ];
 
-/// Cardholder Capability Container (CCC) Identifier
+/// Cardholder Capability Container (CCC) Identifier card ID
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
-pub struct CCCID(pub [u8; YKPIV_CCCID_SIZE]);
+pub struct CccCardId(pub [u8; YKPIV_CCCID_SIZE]);
+
+/// Cardholder Capability Container (CCC) Identifier
+#[derive(Copy, Clone)]
+pub struct CCC(pub [u8; YKPIV_CCC_SIZE]);
+
+impl CCC {
+    /// Return CardId component of CHUID
+    pub fn cccid(&self) -> Result<CccCardId, Error> {
+        let mut cccid = [0u8; YKPIV_CCCID_SIZE];
+        cccid.copy_from_slice(&self.0[CCC_ID_OFFS..(CCC_ID_OFFS + YKPIV_CCCID_SIZE)]);
+        Ok(CccCardId(cccid))
+    }
 
-impl CCCID {
```

Comment:
Why not `CCCID`? I could see the containing module being renamed to `ccc` so it's `ccc::Id` perhaps.

Reply:
Mostly because the yubico-piv-tool returns the entire Cardholder Capability Container object and labels it CCC. The cccid that named the file is a component of this larger piece (like the change to the CHUID struct in an earlier PR).